.gitignore

```diff
@@ -1 +1,23 @@
+.DS_Store
+node_modules
+/dist
 .idea
+
+# local env files
```

Comment:
Remove comments, please.

Reply:
These were generated by setting up Vue CLI , I will delete all the comments as You wish.